### DISCUSSION TITLE
Error installing genesis package to use Doc2Vec

### DISCUSSION THREAD

**Discussion by Morle2104** [asreview/asreview#1950] [Q&A] [answered]
Dear all, 

I want to set up a project with Doc2Vec as the feature extractor and Random Forest as the classifier. I know I need the gensim package to use Doc2Vec. I used the pip install asreview[gensim] in my command prompt and I tried to install it but when I then go to ASReview and want to use the Doc2Vec, it gives me an error during the 'warm up' phase of setting up the project. "**_ImportError: Install gensim package to use Doc2Vec_**". 
I also installed the tensorflow package for the Random Forest classifier, and that worked fine. I checked on the location on my laptop, and I find the tensorflow package there but not the gensim one. 

Any ideas what I might be doing wrong?

Many thanks!
Best wishes!

**Comment by timovdk** [comment]
Hi @Morle2104, could you try to install gensim using the following command `pip install gensim`?

**Reply by Morle2104** [reply, marked as answer]
Thank you for your quick response! I think I just found the answer to the problem here: https://github.com/asreview/asreview/issues/1757#issue-2331077259 :-)
